homeassistant/components/enigma2/media_player.py

```diff
@@ -136,7 +143,15 @@ def __init__(self, name: str, device: OpenWebIfDevice, about: dict) -> None:
 
     async def async_turn_off(self) -> None:
         """Turn off media player."""
-        await self._device.turn_off()
+        try:
+            await self._device.turn_off()
+        except ServerDisconnectedError as err:
```

Comment:
Are these exceptions new in the bumped library? Otherwise please move it to a separate PR.

Reply:
Okay, I have reverted that change and will put it in a separate PR. Now it is only the neccessary changes to fix #107390 and the library bump to do that.